build/build.js

```diff
@@ -0,0 +1,112 @@
+const rollup = require('rollup');
+const typescript = require('rollup-plugin-typescript2');
+const rimraf = require('rimraf');
+
+const LIB_NAME = 'aurelia-store';
+const cacheRoot = '.rollupcache';
+const externalLibs = [
+  'aurelia-dependency-injection',
```

Comment:
how do you define the list of the external deps? Isn't this meant to replicate the package.json dependencies? I don't know where the Event-Aggregator is of need. If so, can't we just require the section from package.json. That reduces maintenance efforts if new deps are added

Reply:
The whole array can be omitted, It's just a mean to reduce the warning. By this, we signal rollup to not care about those libs, same with its being unable to resolve it (which will result in warning). The array is what I extracted out from all of the imports in src

Reply:
well in this case loading and pumping the deps section of package.json should pretty much do that right?